docs/UPGRADE.md

```diff
@@ -0,0 +1,20 @@
+# HOW TO UPGRADE PARTICIPA.SOMENERGIA.COOP
+
+## Upgrade notes
+
+Update your Gemfile:
+```
+gem "decidim", "0.XX.X"
+gem "decidim-dev", "0.XX.X"
+```
+
+Run these commands to upgrade and make sure you get all the latest migrations:
+```
+bundle update decidim
```

Comment:
Great idea to add this document @aitorlb 
Just note that this should be executed locally, and you are not metioning the git commands . I would add the git commands and explain it here with a notice.
I'll also add a new section for what should be executed in production that is only the `db:migrate`.
```suggestion
# to be executed locally on the developer's machine
bundle update decidim
bin/rails decidim:upgrade
git add db/migrations
git commit db/migrations
git push
bin/rails db:migrate

# to be executed in the deployment environments
bin/rails db:migrate
```

Reply:
It was @isaacmg410 idea. Thanks for the tips.